site/docs/samples/chaining-sample.md

```diff
@@ -0,0 +1,54 @@
+# FybrikModule-chaining sample
+
+This sample shows how to implement a use case where based on the data source and governance policies, the Fybrik manager determines that it must deploy two FybrikModules to allow a workload access to a dataset.
```

Comment:
comma after where
Add at the end of the sentence.  One FybrikModule handles reading the data and the second does the data transformation.  Data is passed between the FybrikModules without writing to intermediate storage.